recipes/portalocker/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```

Comment:
Can we get rid of the `bld.bat` and `build.sh` in favor of the `script` entry in the `meta.yaml` files:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L29